lib/commands.sh

```
@@ -163,21 +298,49 @@ cmd_policy() {
 }
 
 usage() {
+  local cmd="$(basename "$0")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)